students/ZackConnaughton/session03/slicing_lab.py

```diff
@@ -0,0 +1,61 @@
+
+
+def exchange_first_last(seq):
+    """
+    Sequence entry returns a copy of the sequence with the first and last items
+    swapped
+    """
+    return seq[-1:] + seq[1:-1] + seq[:1]
+
+
+def remove_every_other(seq):
+    """
+    Sequence entry returns a copy of the sequence with every other item removed
+    """
+    return seq[0:len(seq):2]
+
+def remove_first_and_last(seq, num=4):
+    """
+    Sequence entry returns a copy of the sequence with a number of items
+    removed from the start and the end. Returns every other item in the
+    remaining.
+    num - The number of items to remove, default is 4
+    """
+    return remove_every_other(seq[num:-num])
```

Comment:
nice use of existing function, though in this case:

`seq[num:-num:2]`

would do it too.